index.html

```diff
@@ -66,5 +66,173 @@ <h2 class="lets-connect">LET'S CONNECT</h2>
       </ul>
     </div>
   </section>
+  <section class="works-skills">
+    <section class="works">
+    <div class="works-one">
+      <div class="works-top-one">
+      <a href="#" class="snapshoot-one"><img src="./images/SnapshootPortfolio.svg" alt="portfolio-one"></a>
+      </div>
+      <div class="project-one-info">
+      <h3>Tonic</h3>
+      <ul class="project-details-one">
+          <li>Canopy</li>
+          <li>BackEnd Dev</li>
+          <li>2015</li>
+      </ul>
+      <div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div>
+      <ul class="languages-works-one">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      </div>
+      <div>
+        <h6 class="project-link-one">See Project</h6>
+      </div>
+      </div>
+    </div>
+    <div class="works-two">
+      <div class="works-top-two">
+      <a href="#" class="snapshoot-two"><img src="./images/SnapshootPortfolio2.svg" alt="portfolio-two"></a>
+      </div>
+      <div class="project-two-info">
+      <h3>Multi-Post Stories</h3>
+      <ul class="project-details-two">
+          <li>Canopy</li>
+          <li>BackEnd Dev</li>
+          <li>2015</li>
+      </ul>
+      <div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div>
+      <ul class="languages-works-two">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+    </div>
+      <h6 class="project-link-two">
+        See Project
+      </h6>
+      </div>
```

Comment:
- [ ] The previous reviewer mentioned updating the design to match the original one so, please fix your button by adding the border to it also this should be a link `a` or a `button` tag because it will show you the details of the given project, the height is bigger than the original design as well, Kindly fix that to improve your project design